tox.ini

```diff
@@ -25,5 +25,8 @@ deps = flake8
 commands =
   flake8
 
+[flake8]
+per-file-ignores = harvester_e2e_tests/apis/test_support_bundle.py: E501
```

Comment:
In my opinion, we should not exclude single file for _E501_, it would break the window then other files would like to be the one.
(In most cases, you would still have ways to fit _E501_)

Reply:
I'll look to see what we can refactor to shrink the line length. If we're open to it, I'd think that we could expand the limit for the 'maximum' line length to something like 120 characters.  But after I make a few changes, we may be able to remove the exclusion because the file will meet all E501 requirements :smile:

Reply:
I'm thinking even after some cleanup all lines are less than 90 characters:
```
pep8 run-test: commands[0] | flake8
./harvester_e2e_tests/apis/test_support_bundle.py:129:80: E501 line too long (85 > 79 characters)
./harvester_e2e_tests/apis/test_support_bundle.py:130:80: E501 line too long (89 > 79 characters)
```
Instead of:
```
per-file-ignores = harvester_e2e_tests/apis/test_support_bundle.py: E501
```
I'll be pushing a change that tells Flake8 we allow up to 99 - I'm willing to change that though if we feel that's too much.  I've added that configuration in the `tox.ini`.

Though, via [Flake8 Rules](https://www.flake8rules.com/rules/E501.html) it seems that a maximum length of 99 characters is excepted.

From https://peps.python.org/pep-0008/#maximum-line-length :
> Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the line length limit up to 99 characters, provided that comments and docstrings are still wrapped at 72 characters.

In this case it's not the 'docstrings' that are longer, it's our RegEx Pattern's length that is causing the `tox -e pep8` issues.

Reply:
I'm up for expand to 99.